lib/edge_rider/collect_ids.rb

```diff
@@ -51,5 +51,19 @@ def collect_ids
 
     ::Integer.send(:include, Integer)
 
+    module String
+
+      def collect_ids
+        if match(/\A\d+\z/)
+          [to_i]
+        else
+          raise Uncollectable, "Cannot collect an id from #{inspect}"
+        end
+      end
+
+    end
+
+    ::String.send(:include, String)
```

Comment:
While this looks sane, expectations will vary about what `#collect_ids` does on strings like `"1,2,3"`. Since it is not required for your initial request, please drop this.